### PR TITLE
Update score version workflow to also upload tiles

### LINKED ISSUE #2035: Add github action for creating new score versions in s3
Before @esfoobar-usds left, he added score versioning in #1453. However, we need one more github action to actually create where these versions go. He described it as follows:

We need to add "generate version" which will be like Generate Score but it's modeled after deploy_be_to_staging and takes two variables:
- J40_VERSION_DATESTRING
- J40_VERSION_LABEL_STRING

The difference from deploy_be_staging, it'll put the data in data/score/VERSION instead of storing the score by hash.

### PR DESCRIPTION
**Pull request by mattbowen-usds**
Closes #2035

### CODE REVIEW

**Comment by vim-usds** [comment]
@mattbowen-usds and @KameronKerger - 

Seems like the beta folders worked nicely and the beta files appear to be on S3:

Some questions:

1. How does one validate that these files are indeed for beta and not 1.0?
2. The pic below shows the link for the "Data & documentation" however I'm not sure which one should be for the "Shapefile & codebook"


Screen Shot 2022-10-27 at 9 53 18 AM

**Comment by mattbowen-usds** [comment]
1. It turns out I was confused what is beta and what is 1.0. So these are for 1.0. You can check by seeing if the field "Identified as disadvantaged due to tribal overlap" field is in the downloads --- that exists only in narwhal.
2. Good catch! I needed to rearrange so that file is generated BEFORE the upload to s3, not after. Testing a fix there now.

File: data/data-pipeline/data_pipeline/files/README-version-1.0.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Draft, Deliberative and Pre-Decisional
```
[the remark this answers is not in the document]
Comment (vim-usds) [reply data/data-pipeline/data_pipeline/files/README-version-1.0.md:1]: Thanks! The content does look right

**Review by vim-usds — APPROVED** [review]
Thanks Matt for expediting this!

**Review by vim-usds — APPROVED** [review]

**Pull request merged**